hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopFsGenerator.java

```diff
@@ -63,6 +63,12 @@ public class HadoopFsGenerator extends BaseFreonGenerator
       defaultValue = "4096")
   private int copyBufferSize;
 
+  @Option(names = {"-sync"},
```

Comment:
Please avoid single-dash long options.

```suggestion
  @Option(names = {"--sync"},
```